### PR TITLE
Update javascript cheats

### PR DESCRIPTION
**Pull request by toan2406**
Update javascript cheat with `yarn` and run script command

### CODE REVIEW

File: cheats/javascript.cheat [hunk 1]
```diff
@@ -21,6 +21,38 @@ npm install <package_name> --save-dev
 # install globally a specified package
 npm install <package_name> -g
 
+# run a script
+npm run <script>
+
+$ script: cat package.json | python -c "import sys, json; print('\n'.join(json.load(sys.stdin)['scripts'].keys()))"
```
Comment (OriR) [inline cheats/javascript.cheat]:
```
node -p "Object.keys(require('./package.json').scripts).join('\n')"
```

If you're searching for `node` snippets, you'll probably have `node`, not necessarily `python` 😃

Comment (toan2406) [reply cheats/javascript.cheat]: lol, that makes sense. Let me update it

Comment (OriR) [reply cheats/javascript.cheat]: Also, you don't need the `cat` now, the `require('./package.json')` part reads the package.json file for you 🙃

Comment (toan2406) [reply cheats/javascript.cheat]: Thanks. I forgot that

**Pull request merged**